testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AvailableJavaHomes.java

```diff
@@ -178,7 +171,7 @@ public static List<Jvm> getSupportedWorkerJdks() {
      * Get a JDK for each major Java version that is not able to run the Gradle daemon, if available.
      */
     public static List<Jvm> getUnsupportedDaemonJdks() {
-        return getJdksInRange(Range.lessThan(SupportedJavaVersions.MINIMUM_DAEMON_JAVA_VERSION));
+        return getJdksInRange(Range.closedOpen(SupportedJavaVersions.MINIMUM_WRAPPER_JAVA_VERSION, SupportedJavaVersions.MINIMUM_DAEMON_JAVA_VERSION));
```

Comment:
```suggestion
        return getJdksInRange(Range.closedOpen(SupportedJavaVersions.MINIMUM_CLIENT_JAVA_VERSION, SupportedJavaVersions.MINIMUM_DAEMON_JAVA_VERSION));
```

Would prefer to use the client version here (even though they are the same number). In practice the vast majority of our tests run though the client and only a very small number runs through the wrapper. 

Using the client version IMO better represents our intentions that we don't run tests below the client minimum, since otherwise the client would fail